packages/react-sdk/docusaurus/docs/React/10-advanced/11-keyboard-shortcuts.mdx

```diff
@@ -0,0 +1,98 @@
+---
+id: keyboard-shortcuts
+title: Keyboard shortcuts
+---
+
+## Introduction
+
+In this guide, you'll learn how to apply keyboard shortcuts to your Stream Video application. Some users find it much easier to use their keyboard instead of reaching for their mouse/touchpad and having to find an unmute button to be then able click to unmute. In this example we'll be implementing `Ctrl + d` (or `⌘ + d` on Mac) and `Ctrl + e` (or `⌘ + e` on Mac) shortcuts to mute/unmute your video and audio.
+
+### Define custom `useKeyboardShortcuts` hook
+
+As this hook will introduce keyboard shortcuts related to [`CallControls`](../03-ui-components/call/call-controls.mdx) (mute/unmute audio) we should ideally invoke it in the same component to group logical parts together for easier debugging later. The same goes for your other keyboard shortcuts you might introduce to your application. Learn how to adjust your `CallControls` component in [the Replacing Call Controls guide](../06-ui-cookbook/02-replacing-call-controls.mdx).
+
+```ts
+const isMacOS = () => !!navigator.userAgent.match(/(Mac\s?OS)/g)?.length;
+
+export const useKeyboardShortcuts = () => {
+  const { toggleAudioMuteState } = useToggleAudioMuteState();
+  const { toggleVideoMuteState } = useToggleVideoMuteState();
```

Comment:
Can you add the import statements for these hooks? I think our customers would appreciate it :)

Reply:
Nice catch, I'll add those. 👍